Lib/test/test_zipfile.py

```diff
@@ -549,6 +549,28 @@ def test_add_file_before_1980(self):
         with zipfile.ZipFile(TESTFN2, "w") as zipfp:
             self.assertRaises(ValueError, zipfp.write, TESTFN)
 
+    def test_add_file_before_1980_no_strict_timestamps(self):
+        # Set atime and mtime to 1970-01-01
+        os.utime(TESTFN, (0, 0))
+        with zipfile.ZipFile(TESTFN2, "w") as zipfp:
+            zipfp.write(TESTFN, strict_timestamps=False)
+            zinfo = zipfp.getinfo(TESTFN)
+            self.assertEqual(zinfo.date_time, (1980, 1, 1, 0, 0, 0))
```

Comment:
I would help to also make sure that you can add error when using strict_timestamps=True here.